terra_opp/models.py

```diff
@@ -63,6 +63,7 @@ class Viewpoint(BaseLabelModel):
     )
     properties = JSONField(_("Properties"), default=dict, blank=True)
     related = GenericRelation("datastore.RelatedDocument")
+    viewpoint_number = models.PositiveIntegerField(_("Viewpoint number"), unique=True)
```

Comment:
This is the viewpoint model so I think we don't need the "viewpoint prefix". Also number is too generic so may be something like `identifier` ?

Reply:
Done.